chall04/jnovotny.py

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/python3
+#Python 3.7.3
+
+import sys
+
+def map_error_exit(msg):
+	print(f"Error occured: {msg}")
+	exit()
```

Comment:
I really love the new format of python3!

I would suggest using sys.exit(arg) instead of creating your own function. 
https://docs.python.org/2/library/sys.html#sys.exit